engines/bastion/lib/bastion/engine.rb

```diff
@@ -46,6 +46,7 @@ class Engine < ::Rails::Engine
     initializer 'bastion.register_plugin', :before => :finisher_hook do
       Foreman::Plugin.register :bastion do
         requires_foreman '>= 1.15'
+        path File.join(Katello::Engine.root, 'engines', 'bastion')
```

Comment:
Could we just drop bastion as a Foreman plugin all together?

Reply:
very likely, i will give it at try

Reply:
seems to work fine with latest updates